float32 pressure
float32 loadCell
float32 temperature
float32 kalmanX
float32 kalmanY
float32 accelX
float32 accelY
float32 accelZ
float32 gyroX
float32 gyroY
float32 gyroZ

